fastpair/test/test_fastpair.py

```diff
@@ -277,23 +273,23 @@ def test_iter(self, PointSet):
         assert fp[ps[0]].neigh in set(ps)
 
         try:
-            myitem = fp[(2, 3, 4)]
+            fp[(2, 3, 4)]
         except KeyError as err:
             print(err)
 
         fp[ps[0]] = fp[ps[0]].neigh
         try:
-            fp[(2, 3, 4)] = fp[ps[0]].neigh
+            fp[ps[0]].neigh  # noqa: B018
```

Comment:
this is here just to ensure it doesn't throw. I'm sure there are better/more idiomatic ways to do this

Reply:
> So we could either turn those back on, or just drop res and neigh I suspect.

I'll try to turn those back on and ....

> this is here just to ensure it doesn't throw. I'm sure there are better/more idiomatic ways to do this

maybe add some kind of `assert fp[ps[0]].neigh` for the something or even simply `assert hasttr(fp[ps[0]], "neigh")`